src/status_im2/common/mute_chat_drawer/view.cljs

```diff
@@ -0,0 +1,67 @@
+(ns status-im2.common.mute-chat-drawer.view
+  (:require [utils.i18n :as i18n]
+            [quo2.core :as quo2]
+            [react-native.core :as rn]
+            [status-im2.constants :as constants]
+            [utils.re-frame :as rf]
+            [status-im2.common.mute-chat-drawer.style :as style]))
+
+(defn hide-sheet-and-dispatch
+  [event]
+  (rf/dispatch [:bottom-sheet/hide])
+  (rf/dispatch event))
+
+(defn mute-chat-drawer [chat-id accessibility-label]
```

Comment:
I couldn't find any usage of this function `mute-chat-drawer`, could you elaborate?

Reply:
This is the community channels list of muting options .. Will use it in another PR that is supposed to use this feature

Reply:
I still find weird to merge code that has zero usages @ibrkhalil. How do you know it works and renders appropriately if nothing renders it?

Reply:
I already rendered it and removed the usage till the other one is merged

Reply:
All right, that's more acceptable, thanks. Next time please add a Github comment or something explaining you need to merge something like this because it depends on some other PR. I generally wouldn't approve code that has no usages, I think it's just something to avoid.